.readthedocs.yml

```diff
@@ -4,7 +4,7 @@ build:
     image: latest
 
 conda:
-    environment: ci/requirements/readthedocs.yml
+    environment: requirements/ci/readthedocs.yml
```

Comment:
@tkknight This should be enough for `readthedocs` to find the associated `yml` file.